app/src/main/java/io/qtalk/qgamejsinterfacetester/MainActivity.kt

```diff
@@ -72,19 +77,24 @@ class MainActivity : AppCompatActivity() {
             .negativeButton(text = "Test URL") {
                 openTestUrl()
             }
-            .checkBoxPrompt(text = "Make a preview call", onToggle = null)
-            .input(prefill = getUrlFromPrefOrIntent()) { materialDialog, charSequence ->
-                val urlToOpen = charSequence.toString()
+            .listItemsSingleChoice(R.array.interactionTypes, initialSelection = 1, waitForPositiveButton = false){ dialog: MaterialDialog, index: Int, text: String ->
```

Comment:
[optional] Missing formatting rules:
```suggestion
            .listItemsSingleChoice(R.array.interactionTypes, initialSelection = 1, waitForPositiveButton = false) { dialog: MaterialDialog, index: Int, text: String ->
```

Reply:
😅 I'll make the other small changes.